scripts/update-showcase.mjs

```diff
@@ -435,6 +435,17 @@ const scraper = new ShowcaseScraper({
 		"https://souto.tk",
 		"https://unwrapped.studio/",
 		"https://gdscyu.com/",
+		"https://ahuja-lab.in/",
+		"https://roudstudio.com/",
+		"https://oengi.com/",
+		"https://kireerik.github.io/refo/",
+		"https://aiaman.web.app/",
+		"https://zenorocha.com/",
+		"https://www.rmmanfredi.com/",
```

Comment:
This looks to have Astro hints
![image](https://github.com/withastro/astro.build/assets/64310361/139f856a-2b64-42fa-86d7-1aa58e283406)
```suggestion
```